[DOCS] Value list exceptions for all rule types (#2562)
elastic/security-docs: docs/detections/detections-ui-exceptions.asciidoc

After creating value lists, you can use `is in list` and `is not in list`
operators to define exceptions.

TIP: You can also use value lists as the <<indicator-value-lists,indicator match index>> when creating an indicator match rule.

[float]
[[manage-value-lists]]
== Create value lists

> When creating a value list for a rule exception, be mindful of the list's size and data type. In general, value list exceptions are supported for all rule types, but there are some limitations for especially large lists or certain data types. The following value list types can _only_ be used with custom query, machine learning, and indicator match rule types:

Reviewer: Flipping this to active voice. 
```suggestion
When creating a value list for a rule exception, be mindful of the list's size and data type. In general, all rule types support value list exceptions, but there are some limitations for especially large lists or certain data types. The following value list types can _only_ be used with custom query, machine learning, and indicator match rule types:
```

+
[NOTE]
=======
* An exception defined by a value list must use `is in list` or `is not in list` in all conditions.
* Wildcards are not supported in value lists.

> * If a value list can't be used due to <<manage-value-lists,size or data type>>, it will appear unavailable in the *Value* menu.

Reviewer: ```suggestion
* If a value list can't be used due to <<manage-value-lists,size or data type>>, it'll be unavailable in the *Value* menu.
```
